src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -107,6 +107,16 @@
     private transient boolean closed;
 
     /**
+     * Maximum number of log lines for a stack trace
+     */
+    public static final int DEFUALT_MAX_LENGTH = 1000;
+
+    /**
+     * Maximum number of log lines for a stack trace
```

Comment:
@dean-e-clark according to http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html, first sentence should alsways end with period